test/src/main/java/org/apache/accumulo/test/ShellIT.java

```diff
@@ -400,7 +400,7 @@ public void execFileTest() throws IOException {
     Shell.log.debug("Starting exec file test --------------------------");
     shell.config("--config-file", config.toString(), "-u", "root", "-p", getRootPassword(), "-zi",
         getCluster().getInstanceName(), "-zh", getCluster().getZooKeepers(), "-f",
-        "src/main/resources/shellit.txt");
+        "src/main/resources/shellit.shellit");
```

Comment:
Why did you need to rename this?

Reply:
Because the filename extension, `.txt` has a specific format for the license header, built-in to the plugin, and the choice was either rename this test file (which I did), or override the built-in rule for *ALL* `.txt` files, which I didn't think was a good idea.

Reply:
Thanks for the explanation.